Mixpanel/Mixpanel.h

```diff
@@ -64,6 +64,13 @@ extern NSString *const MPNotificationTypeTakeover;
  */
 @property (atomic, readonly, strong) MixpanelPeople *people;
 
+/*!
+ The distinct ID of the current user that can be called from any queue.
+
+ See the documentation for distinctId below for more information.
+ */
+- (NSString *)safeDistinctId;
```

Comment:
Obviously there should only be one public distinct ID but I just added this one to minimize changes for initial feedback.